lib/evmone/instructions.cpp

```diff
@@ -629,17 +599,15 @@ void op_blockhash(execution_state& state, instr_argument) noexcept
     auto upper_bound = state.host.get_tx_context().block_number;
     auto lower_bound = std::max(upper_bound - 256, decltype(upper_bound){0});
     auto n = static_cast<int64_t>(number);
-    auto header = evmc_bytes32{};
+    auto header = evmc::bytes32{};
```

Comment:
Seems to be a somewhat irrelevant change?

Reply:
Removed.